### PR TITLE
CB-15422 add wait for CLUSTER_UNREACHABLE state

### PR DESCRIPTION
**Pull request by sodre90**
See detailed description in the commit message.

### CODE REVIEW

**Review by gergopapi2 — APPROVED** [review]

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxRepairTests.java [hunk 1]
```diff
@@ -155,6 +155,8 @@ public void testSDXMediumDutyRepair(TestContext testContext) {
                     return testDto;
                 })
                 .awaitForHostGroups(List.of("gateway", "idbroker"), InstanceStatus.DELETED_ON_PROVIDER_SIDE)
+                .await(SdxClusterStatusResponse.CLUSTER_UNREACHABLE, Set.of(SdxClusterStatusResponse.NODE_FAILURE),
```
Comment (foldik) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxRepairTests.java:158]: Is it the only place where we have this kind of flakiness?

**Review by lnardai — APPROVED** [review]

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxRepairTests.java (hunk 1 above)
Comment (sodre90) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxRepairTests.java:160]: good catch, thanks, I added it to one more testcase

Comment (topolyai5) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxRepairTests.java:160]: You didn't found other test case?

Comment (sodre90) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxRepairTests.java:160]: yes, it was in the same class

**Pull request merged**